README.md

```diff
@@ -8,19 +8,29 @@ doesn't handle is posting your files back to an issue for review.
 
 ## Installation
 
-Install dependencies with Composer:
+Dorgflow is designed to be installed globally.
+
+1. Either clone the repository or download a release and place it anywhere you
+like.
+
+2. Install dependencies with Composer:
 
     $ composer install --no-dev
 
 (The --no-dev option omits the packages that are needed for testing.)
 
-Then either:
+3. To make the command accessible, either:
   - symlink the file dorgflow into a folder that's in your path.
   - set the root folder of this repository into your path.
 
-### Optional: For composer driven Drupal projects
+### Additional: For composer driven Drupal projects
 
-If you're about to use Dorgflow in the context of a composer driver Drupal project, where you e.g. want to use version 8.x-3.1 of a particular module for staging and the production server, you may easily run into trouble setting up your environment for drupal.org contributions controlled by dorgflow. For those of you there is a composer plugin available which helps you manage those environments: https://packagist.org/packages/lakedrops/dorgflow
+If you're about to use Dorgflow in the context of a composer driver Drupal
+project, where you e.g. want to use version 8.x-3.1 of a particular module for
+staging and the production server, you may easily run into trouble setting up
+your environment for drupal.org contributions controlled by dorgflow. For those
+of you there is a composer plugin available which helps you manage those
+environments: https://packagist.org/packages/lakedrops/dorgflow
```

Comment:
I think this section is still confusing.  Most every D8 project these days is "composer driven".  This implies that if you use composer then you should use the lakedrops/dorgflow package, which isn't the case.  I haven't run into "trouble setting up your environment" and don't even know what this refers to.

Reply:
I agree, but the problem is that I don't really know what it means myself, as I didn't write it.

I *think* that plugin lets you easily have Composer install the contrib modules you specify as git clones rather than just flat files.

But that's not my workflow at all. I usually just move out the composer-managed contrib module folder, symlink in a git clone from my sandbox folder, and then make the changes I want, and then when I'm done, move the composer-managed folder back in! Probably not ideal, but it means I don't have to fight with Composer.

Reply:
I too agree. Maybe it is time to reach out to the maintainers of the `lakedrops/dorgflow` project to ask what advantage there is to using their project verses globally installing `joachim-n/dorgflow`.

Reply:
I emailed @jurgenhaas to see if he would help us out on finishing this issue with some advice since he is the maintainer of the lakedrops/dorgflow composer package. It is hosted on a private Gitlab instance so that is why I emailed him. He does have a Github account. Hopefully we will hear from him soon.

Reply:
Just responded to the issue in this [comment](https://github.com/joachim-n/dorgflow/issues/43#issuecomment-590729854)

Reply:
@jurgenhaas Thanks for updating the documentation for your project `lakedrops/dorgflow`. 

Can you please comment on the proposed section to be added to this project `joachim-n/dorgflow` that talks about how to use your project (Lines 28 - 33 of the README.md)?

Does this sound correct to you? Should it be worded differently? Should this project use a modified version of the new documentation you added to your project? Or should this project just add a link to your projects documentation?